documentation/tutorials/text-search-semantic.md

```diff
@@ -0,0 +1,317 @@
+---
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Ny copyright header: "Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root."